### PR TITLE
change default rankdir and attempt to sort if practical

### PR DESCRIPTION
**Pull request by MTN-RowinAndruscavage**
added "rankdir = LR;"  to make the default template somewhat more compact and scrollable

encourage nodes to be drawn sorted alphanumerically, if practical

### CODE REVIEW

**Comment by willthames** [comment]
This seems like a nice fix, but I'd rather it was an option so that we don't unexpectedly change behaviour for users.

**Comment by MTN-RowinAndruscavage** [comment]
Sounds good!  Made a new "-a ATTRIBUTE" option, with a default of "rankdir=TB;" , and documented how to use it to make different types of graphs from the same default template

File: README.md [hunk 1]
```diff
@@ -16,38 +16,62 @@ pip install ansible-inventory-grapher
...
+You can add the `-a` option to insert a string with graphviz attributes (http://www.graphviz.org/doc/info/attrs.html) to apply to the root level of the graph.  Some fun examples:
+```bash
+# transpose the tree so it grows from left-right instead of top-bottom
+-a "rankdir=LR;"
+
+# circular layout, with group nodes shaded grey
+-a "layout=circo; overlap=false; splines=polyline;\
+  node [ style=filled fillcolor=lightgrey ]"
+
+# OCD
```
Comment (willthames) [inline README.md]: Please can you use a different description here? I don't want this to be seen as making light of people with a genuine disorder.

Comment (MTN-RowinAndruscavage) [inline README.md]: Good call!

**Comment by willthames** [comment]
Looks good - would you prefer to squash yourself, me to squash, or trust github's squashing interface?

There shouldn't be any merge commits, and small changes that fix a minor issue with a previous commit aren't worth retaining in the commit history.

There's probably a range of possible number of commits here - should definitely be smaller than 9, but doesn't have to be as small as 1.

File: README.md [hunk 2]
```diff
@@ -16,38 +16,62 @@ pip install ansible-inventory-grapher
...
 Using the example inventory in https://github.com/willthames/ansible-ec2-example,
 we can generate the dot files for two of the example web servers using:
+```bash
+bin/ansible-inventory-grapher -i ../ansible-ec2-example/inventory/hosts \
+  prod-web-server-78a prod-web-server-28a \
+  -d test --format "test-{hostname}.dot"
 ```
-bin/ansible-inventory-grapher -i ../ansible-ec2-example/inventory/hosts prod-web-server-78a prod-web-server-28a -d test --format "test-{hostname}.dot"
+
+You can add the `-a` option to insert a string with graphviz attributes (http://www.graphviz.org/doc/info/attrs.html) to apply to the root level of the graph.  Some fun examples:
```
Comment (willthames) [inline README.md]: can hyperlink this properly i.e. `[graphviz attributes](http://www.graphviz.org/doc/info/attrs.html)`

**Comment by MTN-RowinAndruscavage** [comment]
OK, squashed everything into a new PR #22

File: README.md (hunk 2 above)
Comment (MTN-RowinAndruscavage) [inline README.md]: Oh I thought you preferred leaving the full link, so you can see the URL without having to mouseover.  But if you'd like to convert everything over, we can do that in a later PR.

**Pull request closed without merging**